loadable_apps/loadable_sample/wifiapp/wifiapp.c

```diff
@@ -83,35 +83,36 @@ int wifiapp_main(int argc, char **argv)
 #ifdef CONFIG_EXAMPLES_LOADABLE_MANUAL_TEST
 	while (is_testing) {
 		display_test_scenario();
-		ch = getchar();
-		switch (ch) {
+		while ((ch = getchar()) != EOF) {
```

Comment:
The `ch` is char type so that after getting the return value of getchar, it can't compare with EOF.

Reply:
I changed the `ch` type to int. Thanks.